### PR TITLE
Restore exceptions logic on read configuration

### PR DESCRIPTION
**Pull request by anisimkov**
S223-004 T707-002

Ignore all exceptions on reading configuration at elaboration code.
Restore exceptions  propagation when load configuration by user.

### CODE REVIEW

File: src/core/aws-config.adb [hunk 1]
```diff
@@ -583,16 +582,17 @@ package body AWS.Config is
...
-   -- Read_If_Present --
-   ---------------------
+   --------------------
+   -- Read_Or_Ignore --
+   --------------------
 
-   procedure Read_If_Present (Filename : String) is
+   procedure Read_Or_Ignore (Filename : String) is
    begin
-      if Directories.Exists (Filename) then
```
Comment (TurboGit) [inline src/core/aws-config.adb:592]: I don't agree with this change as it uses exception as control flow.

Comment (TurboGit) [reply src/core/aws-config.adb:592]: We still need to check for Directory.Exists() and catch all other exceptions if needed.

Comment (TurboGit) [reply src/core/aws-config.adb:592]: Apart from that I'm ok with this change. We hide internal exception when reading ini file but still raise exception for explicit call to Read() from user code. Seem a good compromise.

Comment (anisimkov) [reply src/core/aws-config.adb:592]: Ok. I'll restore check file existence inside of Read_Or_Ignore.

**Pull request merged**